schools/cornell/enroll.go

```diff
@@ -0,0 +1,312 @@
+package cornell
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"net/http"
+	"net/http/cookiejar"
+	"net/url"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/MyHomeworkSpace/api-server/data"
+	"github.com/MyHomeworkSpace/api-server/email"
+	"github.com/MyHomeworkSpace/api-server/schools"
+	"github.com/PuerkitoBio/goquery"
+)
+
+var apiKeyRegexp = regexp.MustCompile(`"key":"(?P<Key>[^"]*)`)
+var nameRegexp = regexp.MustCompile(`login.*"name":"([^"]*)`) //kinda hacky, I should fix this
+
+func (s *school) Enroll(tx *sql.Tx, user *data.User, params map[string]interface{}) (map[string]interface{}, error) {
+	netidRaw, ok := params["netid"]
```

Comment:
For consistency, can you make this "username"? I think at some point we're going to want to merge parts of the schools' login pages, and it will be easier if they take roughly the same parameters.